users/whitmann/index.md

```diff
@@ -25,50 +25,75 @@ My current research involves solving the radiative transport equation for phonon
 
-* **N. WHITMAN**, B. COX, "Characterization of the Pulsed Radiation Environment in FREC-II at the ACRR", *Trans. Am. Nucl. Soc*, **115**, 1074-1077 (2016).
+* **N. WHITMAN**, and B. COX, "Characterization of the Pulsed Radiation Environment in FREC-II at the ACRR", *Trans. Am. Nucl. Soc*, **115**, 1074-1077 (2016).
 
-* **N. WHITMAN**, P. MCCLURE, D. POSTON, "Modeling and Analysis of the BUK/RES-5 Fast Reactor Using MCNP”. *LA-UR-15-25998*, (2015).
+* **N. WHITMAN**, P. MCCLURE, and D. POSTON, "Modeling and Analysis of the BUK/RES-5 Fast Reactor Using MCNP”. *LA-UR-15-25998*, (2015).
 
 ***
 
 ## Presentations
-* ***International Conference on Transport Theory (ICTT)*** **Pierre and Marie Curie Campus (PMCC) at Sorbonne University** *Paris, France* September 22-27, 2019.
+* ***American Nuclear Society (ANS) Mathematics and Computational (M&C) Division*** *Raleigh, North Carolina* April, 2021
+
+* ***American Nuclear Society (ANS) Winter Meeting 2019*** *Washington, D.C.* November, 2019
+
+* ***International Conference on Transport Theory (ICTT)*** **Pierre and Marie Curie Campus (PMCC) at Sorbonne University** *Paris, France* September 22-27, 2019
 
-* ***Institute of Nuclear Materials Management (INMM)*** **Nuclear Materials Science, Processing
-and Signature Discovery Workshop** *Richland, Washington* May, 2018
+* ***Institute of Nuclear Materials Management (INMM)*** **Nuclear Materials Science, Processing and Signature Discovery Workshop** *Richland, Washington* May, 2018
 
 * ***Pacific Northwest National Laboratory (PNNL)*** **National Security Directorate's Student Poster Presentation** *Richland, Washington* Summer 2017
 
-* ***American Nuclear Society (ANS) Winter Meeting*** **Student Design Competition** *Las Vegas, Nevada* November, 2016
+* ***American Nuclear Society (ANS) Winter Meeting 2016*** **Student Design Competition** *Las Vegas, Nevada* November, 2016
 
 * ***Los Alamos National Laboratory (LANL)*** **W-Division Student Poster Presentation** *Los Alamos, New Mexico* August, 2016
 
 ***
 
 ## Work Experience
-* ***TerraPower, LLC*** , Mentors: Jacob Hader & Oded Doron, *June - September, 2019*
-  * **Nuclear Design Group - Shielding Methods and Analysis**
+* ***Los Alamos National Laboratory (LANL)***
+  * **XCP-3: Monte Carlo Methods, Codes, and Applications**, Mentors: Travis Trahan (XCP-3) and Jesson Hutchinson (NEN-2), *June - September 2020*
+    * Verification and validation of the Subcritical Reflected alpha-Plutonium (SCRaP) experiments performed at NCERC using the Monte Carlo Application ToolKit (MCATK) software. The SCRaP experiments involved sixteen configurations of an alpha-phase, weapons grade Plutonium sphere surrounded by varying layers of copper and polyethylene reflectors and two Helium-3 multiplicity counters. Analysis included comparisons of MCATK to MCNP and measured values of multiplication factor, multiplication, count rates, and other counting parameters such as Feynman variance-to-mean.
```

Comment:
weapons grade **p**lutonium

Reply:
**h**elium-3

Reply:
I may not have the background, but are multiplication factor and multiplication different thinks? Just checking :)

Reply:
Everything else looks good! I wish there was a better way to make sure images would show up on the reviewers' end of things, but alas.

Reply:
Thanks for the edits - and multiplication can be interpreted as the average length of fission chains and related to multiplication factor by M = 1/k-1

Reply:
Ahh, I see! The more you know.

Reply:
It's looking good to me too. Once those edits are updated any of us can probably merge the request?

Reply:
Yup yup!

Reply:
Believe I fixed this. I will merge.